drills/deploy/README.md

```diff
@@ -0,0 +1,160 @@
+## Purpose of Deploy.rb
+This document for who using Mina gem at https://github.com/mina-deploy/mina
+
+This document describle how to use 1 deploy.rb file for multible environment.
+
+##Install mina gem
+Open gem file, add mina gem in development group
+
+    group :development do
+      gem 'mina'
+    end
+    
+Then, open terminal, install it
+
+    $ bundle install
+
+After installation is finished, run
+
+    $ mina init
+    Created config/deploy.rb.
+
+##Initialize Amazon web services for deploy.
+Contact adminstrator to setting up deploy environment on AWS.
+
+#Setting mina for environment
+    
+    $ mina setup to=your_app_environment
+
+E.g: mina setting for staging environment
+    
+    $ mina setup to=staging
+
+##Setting common config for both Production environment and Staging environment.
+Open config/deploy.rb
+
+    app               = 'your_app_name'
+    production_domain = 'your_app_production_domain'
+    staging_domain    = 'your_app_staging_domain'
+    user              = 'ec2-user'
+    ruby_version      = 'your_app_ruby_version'
+    ruby_gemset       = 'your_app_gemset'
+
+    set :term_mode, nil
+    set :shared_paths, ['config/database.yml', 'log']
+    set :app, app
+    set :repository, "ssh://#{user}@#{domain}/~/git/#{app}.git"
+    set :user, user
+    set :rvm_path, "/usr/local/rvm/bin/rvm"
+
+##Setting individual config for each environment
+
+    stage = ENV['to'] || 'staging'
+    case stage
+      when 'staging'
+        set :branch, 'master'
+        set :domain, staging_domain
+      when 'production'
+        set :branch, 'master'
+        set :domain, production_domain
+      else
+        print_error "Please specify a stage. eg. mina deploy to=production"
```

Comment:
if stage = ENV['to'] || 'staging'
then there will not be an else condition.

I suggest to set stage = ENV['to'] and to throw an error if no stage is set for explicit declaration of environment to push to.